app/views/admin/reports/index.html.haml

```diff
@@ -13,20 +13,32 @@
   %table.table
     %thead
       %tr
-        %th
+        -# %th
```

Comment:
It was leftover from half completed work of Gargron's, and currently shows
as a checkbox with no label or use. Commenting it out for now seemed
prudent.